### DISCUSSION TITLE
New router, feeds no longer update.

### DISCUSSION THREAD

**Discussion by skyhi69** [FreshRSS/FreshRSS#6766] [Q&A] [answered]
Good day everyone, I am retired and still learning networking.

 I set up some docker containers running on my Raspberry Pi and one of them is FreshRSS. Unfortunately, my old router gave up the ghost and I had to move to a new and different model, which resulted in the assigned ip address for my Pi now being different. According to portainer my FreshRSS container is running fine and all my other containers seem to be running, but my FreshRSS feeds have not updated since I replaced my router. Anyone have any ideas on how I can fix the problem? Something I need to change in the settings?

**Comment by Alkarex** [comment]
Hello,
Your FreshRSS container is not supposed to be affected by your Rasperry Pi having a new IP.

What FreshRSS images are you using, more precisely?

A few things to try:

* Update your FreshRSS container to the newest version
* Reboot your Raspberry Pi
* Try to update the feeds manually through the Web interface, and report any error you might see in the logs (including FreshRSS logs accessible from the Web interface)

**Reply by skyhi69** [reply]
Thank you for the reply... I apologize for my ignorance as a late middle-aged person learning new things, but how do I update my container for FreshRSS? (never updated a container before, I take they don't self update?)

 Do I do that with Portainer? Also, what do you mean by what FreshRSS images am I using? 

Update through the web interface? You mean the IP address with the assigned port that I access my container with my Firefox browser? 

I learned about docker and created my Raspberry Pi last year, and at the same time I created a freshRSS container. It worked great till I installed a new router/modem, so it seems likely that triggered things, but I want to work through the troubleshooting to figure it out (and learn some more), I just want to be careful I don't mes things up more! lol

**Reply by Alkarex** [reply]
Yes, you can update FreshRSS from your Portainer Web interface, and also see which FreshRSS image is being used, from the same place.
You are welcome to send a screenshot.

Regarding updating the feeds, I meant for you to try the FreshRSS button
![image](https://github.com/user-attachments/assets/246484d8-fd95-419a-85d1-0dcd82987750)

**Reply by skyhi69** [reply]
Checking the error logs, it is definitely trying to access my old IP address for freshRSS. How do I change that? I can't find where to adjust it. I am also at a loss to figure out using portainer to update my containers. Heck, even in portainer it says there is a new version of portainer but only supplies a link to the information on it and I don't see a way to even update it. 

Here are some screen shots...(fingers crossed they attach!) 
![Screenshot (7)](https://github.com/user-attachments/assets/86d5f9b1-fdba-4dac-8fd9-e0437ddc3a34)
![Screenshot (8)](https://github.com/user-attachments/assets/c6d75619-71a2-4980-8739-748d6cefeea2)
![Screenshot (9)](https://github.com/user-attachments/assets/cb5bafac-f3de-45c9-8274-f9e5744d9d53)
![Screenshot (10)](https://github.com/user-attachments/assets/462d6f95-a2e2-48a6-a775-43d5dcd54b55)

**Reply by Alkarex** [reply]
The `Failed to connect to 192.168.0.127 port 8081` is not related to FreshRSS but to a service that used to run there for your `makefulltextfeed.php`

**Reply by skyhi69** [reply]
I used Full-text RSS from fivefilters.org (used the Craig Titus tech example on YouTube) to help me obtain pictures and such. So I guess that makes sense. It is looking like I will have to rebuild everything unless I can figure out how to edit them. Thanks for helping me narrow it down.

**Comment by skyhi69** [comment]
Also, i hit the refresh button the other day in FreshRSS, and it said it was updating the feeds (but nothing changed). I tried today, and it says there are no feeds to update. However, clearly the problem must be it is still trying to update via my old IP address and I don't see an easy way to update it in FreshRSS.

**Reply by Frenzie** [reply]
Did you try restarting the container?

**Reply by skyhi69** [reply]
I just tried again. I restarted the container and also restarted my Pi. While when I hit update (or refresh icon) on my freshRSS web page, it now says it is updating all the feeds, when it finishes it changes nothing. I checked the logs, and unfortunately it still is using the old IP address. I need to find a way to change it to my new assigned IP address.

**Comment by skyhi69** [comment, marked as answer]
UPDATE: So because I used full-text RSS to create custom feeds for FreshRSS, I have to click on the "gear" icon for each feed in FreshRSS, hit manage, HIGHLIGHT the old IP address and then paste the new IP onto it and that worked! 

Thank you everyone for your help.... I really appreciate it.
